_toc.yml

```diff
@@ -13,6 +13,8 @@ parts:
     title: Scholars 2023
   - file: info/mentors
     title: Mentors
+    sections:
+    - file: info/mentoring_guidelines
```

Comment:
I'd like to change the folder `info` to `impact-scholars`. Will do that in a separate PR. 

I'll merge all the changes in this PR based on the discussion on slack.